sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -511,6 +511,12 @@ class Dataset[T] private[sql](
    */
   def isLocal: Boolean = logicalPlan.isInstanceOf[LocalRelation]
 
+  /**
+   * Returns true if the `DataSet` is empty
+   *
```

Comment:
this would need a 
```
@since 2.4.0
```

Reply:
@mgaido91 Thanks, I modified the comment.